contracts/bonds/WrappedBondDepository.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import "./wETHBondDepository.sol";
+import "../interfaces/IWOHM.sol";
+
+contract WrappedBondDepository is wETHOlympusBondDepository {
+    using SafeERC20 for IERC20;
+
+    address public wOHM;
+
+    constructor(
+        address _OHM,
+        address _principle,
+        address _treasury,
+        address _DAO,
+        address _feed,
+        address _wOHM
+    ) wETHOlympusBondDepository(_OHM, _principle, _treasury, _DAO, _feed) {
+        wOHM = _wOHM;
+    }
+
+    function updateFeed(address _feed) public onlyPolicy {
+        priceFeed = AggregatorV3Interface(_feed);
+    }
```

Comment:
Should use a PriceProvider instead. Just remove it once you have changed to using the abstract BondDepository